src/platform/Darwin/MdnsImpl.cpp

```diff
@@ -446,8 +448,21 @@ CHIP_ERROR GetAddrInfo(void * context, MdnsResolveCallback callback, uint32_t in
         sdCtx->textEntries.push_back(TextEntry{ strdup(key), reinterpret_cast<const uint8_t *>(strdup(value)), valueLen });
     }
 
-    err = DNSServiceGetAddrInfo(&sdRef, 0 /* flags */, interfaceId, kDNSServiceProtocol_IPv4 | kDNSServiceProtocol_IPv6, hostname,
-                                OnGetAddrInfo, sdCtx);
+    DNSServiceProtocol protocol;
```

Comment:
Not problem, just a heads up - spec paragraph [4.12] (ICHIP instances
MUST silently ignore any IPv4 addresses) is eventually going to obsolete this. No one is compliant with that yet, and we don't have a date set as far as I know. Just though you might want a heads up because I missed that when it went in.

Reply:
Fwiw, my understanding is that there are open issues around that part of the spec and it might change.  But yes, right now there's a spec/impl mismatch here.  #7374 at least tracks it.